lib/engine/config/game/g_18_al.rb

```diff
@@ -229,7 +229,7 @@ module G18AL
    "corporations":[
       {
          "sym":"L&N",
-         "name":"Louisville & Nashville Railroad",
+         "name":"Louisville & Nashville Railroad (HO: Birmingham)",
```

Comment:
the historical objective should not be in the name, it can be referenced with the abilitiy

Reply:
@tobymao: Where can the users find information about which is the historical objective for each corporation? In the physical game this is shown on the corporation charters as this is an ability that the corporation has from the beginning of the game and it does not change.

Reply:
Ah, you mean that it should be appended to the corporation name during setup, by using the ability? Or?

Reply:
I will use "Decorator pattern" to add historical objective to the full name.

Reply:
Did not get decorator to work, so instead I added an attribute in corporation (and minor) that can be used to set some extra info. 
Any extra info is now shown as " (...extra info)" after the corporation name.

An improvement could be to add an extra line on the corporation charter to show the extra information. This line could be at the end, maybe.
But that can be done when the next 18xx that need to show corporation information in a similar way.

Reply:
Think this conversation is resolved.